user_guide_src/source/libraries/validation/009.php

```diff
@@ -6,12 +6,7 @@
  *     'contacts' => [
  *        'name' => 'Joe Smith',
  *         'friends' => [
- *             [
- *                 'name' => 'Fred Flinstone',
- *             ],
- *             [
- *                 'name' => 'Wilma',
- *             ],
+ *             'name' => 'Fred Flinstone',
```

Comment:
It is odd that **friends** is not an array.
And this data does not match `contacts.friends.*.name` below.